inventory/environ.py

```diff
@@ -403,6 +404,16 @@ def getASan(vars_scope):
     if vars_scope["splunk"]["asan"]:
         vars_scope["ansible_environment"].update({"ASAN_OPTIONS": "detect_leaks=0"})
 
+def getDisablePopups(vars_scope):
+    """
+    Configure pop-up settings
+    """
+    popups_disabled = os.environ.get("SPLUNK_DISABLE_POPUPS", "")
```

Comment:
Can't you just add this as another line in `overrideEnvironmentVars` below?

Reply:
I'm fine either way, but give me some small tests please :)